pkg/apis/build/v1alpha1/build_validation.go

```diff
@@ -90,9 +95,9 @@ func (bt *BuildSpec) validateTimeout() *apis.FieldError {
 	maxTimeout := time.Duration(24 * time.Hour)
 
 	if bt.Timeout.Duration > maxTimeout {
-		return apis.ErrInvalidValue(fmt.Sprintf("%s should be < 24h", bt.Timeout), "b.spec.timeout")
+		return apis.ErrInvalidValue(fmt.Sprintf("%s should be < 24h", bt.Timeout), "timeout")
 	} else if bt.Timeout.Duration < 0 {
-		return apis.ErrInvalidValue(fmt.Sprintf("%s should be > 0", bt.Timeout), "b.spec.timeout")
+		return apis.ErrInvalidValue(fmt.Sprintf("%s should be > 0", bt.Timeout), "timeout")
```

Comment:
There is a helper that @k4leung4 added for erroring on a bounded value like this.

Reply:
I'll go comb through the code for it, thanks for the heads up

Reply:
I found the method:
https://github.com/knative/pkg/blob/0e41760cea1d1ce16f4cc311c2d6bd3525bb3da7/apis/field_error.go#L341

The one thing I'm seeing is that the `ErrOutOfBoundsValue` method does not exist in the version of `field_error.go` that's contained in my fork's vendored dependencies. Should I upgrade the dependency version?

Reply:
Yeah, you should just have to update the commit in `Gopkg.toml` and run `./hack/update-deps.sh`.